torch/_library/custom_ops.py

```diff
@@ -662,6 +663,22 @@ def backend_select(keyset, *args, **kwargs):
     def __call__(self, *args, **kwargs):
         return self._opoverload(*args, **kwargs)
 
+    def register_vmap(
```

Comment:
Can you test that we support calling register_vmap multiple times? The desired behavior is that it overwrites the previous declaration. We might need to change the call to self._lib.impl to something more lazy.

Reply:
added now.